grunt/helpers/data/Language.js

```diff
@@ -269,9 +272,9 @@ class Language {
     course._latestTrackingId = -1;
     file.changed();
 
-    const blockFileItems = this.getAllFileItems().filter(({ item }) => item._type === 'block');
-    blockFileItems.forEach(({ file, item: block }) => {
-      delete block._trackingId;
+    const fileItems = this.getAllFileItems().filter(fileItem => fileItem.item._type === this.trackingIdType);
```

Comment:
Did you mean to un-destructure here?

Simplify to one loop?
```js
this.getAllFileItems().forEach(({ file, item }) => {
  if (item._type !== this.trackingIdType) return;
  delete item._trackingId;
  file.changed();
});
```
Caveat: untested and might be fundamentally misunderstanding.

Reply:
> Further thought: should grunt tracking-remove not care about types and instead delete tracking IDs from everywhere?

I want to have a bit more control in testing at the moment. Effectively with the default parameter, it works exactly as you describe. I wouldn't expect anyone to need to manually set the --trackingidtype parameter except to review this code.